integreat_cms/cms/constants/poicategory.py

```diff
@@ -0,0 +1,60 @@
+"""
+This module contains constants for POICategory.
+"""
+from django.utils.translation import gettext_lazy as _
+
+DARK_BLUE = "#3700D2"
```

Comment:
Maybe a screen difference, but I see a truly violet here.
Up to you, but I will still suggest:
https://www.colorhexa.com/00008b

Reply:
The colors were given as fixed values from the app team because they depend on the [iconset of the integreat maplibre style](https://github.com/digitalfabrik/integreat-maplibre-style/blob/gh-pages/iconset.json)...
And this page claims this color is in fact a dark blue:
https://chir.ag/projects/name-that-color/#3700D2

Reply:
Okay, I resolve this one